ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -89,6 +89,7 @@ statement
     | CREATE TYPE (IF NOT EXISTS)? identifier AS type                       #registerType
     | DROP TYPE (IF EXISTS)? identifier                                     #dropType
     | ALTER (STREAM | TABLE) sourceName alterOption (',' alterOption)*      #alterSource
+    | ASSERT TOPIC identifier (WITH tableProperties)? timeout?              #assertTopic
```

Comment:
Just to ask... `ASSERT TOPIC` needs to be here to be processed with other DDL commands instead of with `assertStatement` below?  (That's my guess; just validating.)

Reply:
Yup, the ones below are only used by YATT.

Reply:
Are you adding the "NOT" support in this PR or a separate one?

Reply:
It's going in a separate one.